README.md

```diff
@@ -45,24 +45,27 @@ In order to ensure best practices, this working group may work directly with rep
   - [ICU](http://icu-project.org) (C/C++ i18n library used by v8 and Node.js)
 
 ## Current Members
-  + [Adam Miller](https://twitter.com/millea9)
-  + [Antoine Olivier](https://twitter.com/Toinane)
-  + [Ben Michel](https://twitter.com/obensource)
-  + [Benjamin Zaslavsky](https://twitter.com/Ben_Tiriel)
-  + [Dhruv Jain](https://twitter.com/maddhruv)
-  + [Jonathan Cardoso](https://twitter.com/_jonathancardos)
-  + [Laurent Goderre](https://twitter.com/LaurentGoderre)
-  + [Łukasz Szewczak](https://twitter.com/lukaszewczak)
-  + [Rachel White](https://twitter.com/ohhoe)
-  + [Raja Sekar](https://twitter.com/rajzshkr)
-  + [Richard Littauer](https://twitter.com/richlitt)
-  + [Ryo Aramaki](https://twitter.com/geo_vitya)
-  + [Sam Yamashita](https://twitter.com/sota0805)
-  + [Steven R. Loomis](https://twitter.com/srl295)
-  + [Tiago Danin](https://twitter.com/_TiagoEDGE)
-  + [Vanessa Yuen](https://twitter.com/vanessayuenn)
-  + [Volkan Nazmi Metin](https://twitter.com/volemnic)
-  + [Zeke Sikelianos](https://twitter.com/zeke)
+| Name                  | GitHub                        | Twitter                             |
+|:----------------------|:------------------------------|:------------------------------------|
+| Adam Miller           |                               | https://twitter.com/millea9         |
+| Antoine Olivier       |                               | https://twitter.com/Toinane         |
+| Amor                  | https://github.com/amor520    | https://twitter.com/amor90s         |
```

Comment:
I think this would work the same as a link on GitHub rendered markdown if you just use @amor520 instead of putting in the whole link.

Reply:
update 
how does it look now :-D
![image](https://user-images.githubusercontent.com/14037268/37208830-c629e30a-23dd-11e8-894e-9184fa2aa555.png)
![image](https://user-images.githubusercontent.com/14037268/37208840-cb515b2e-23dd-11e8-9d2e-c61ea66fa3dc.png)